src/test/java/org/folio/inventory/dataimport/util/AdditionalFieldsUtilTest.java

```diff
@@ -336,6 +344,69 @@ public void shouldAdd035If001NotEqual003() {
     Assert.assertEquals(expectedParsedContent, parsedRecord.getContent());
   }
 
+  @RunWith(Parameterized.class)
+  public static class OclcFieldNormalizationTest {
```

Comment:
@JavokhirAbdullayev  Is there any way to add more tests for other corner-cases for these new methods in the AdditionalFieldsUtilTest?

Reply:
there should be only one public method, the others should be private, just pushed changes, thanks, nested test class has 4 test cases to check various scenarios

Reply:
added additinal tests